tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -56,21 +52,41 @@ def signin(self, username: str, password: str):
             self.session.close()
             sys.exit()
 
-        response_json: dict = response.json()
-
-        status: str = response_json['data']['signin']['status']
```

Comment:
Changed this slightly as `status` isn't available unless the response was successful.